### ISSUE TITLE
Panaceus bad, Panaceus very bad.

### ISSUE THREAD

**Issue by Bulletfang** [CleverRaven/Cataclysm-DDA#43413]
# Describe the bug

Consuming a Panaceus does not raise stats temporarily by **+2** to everything, but when its effect ebbs away the player is permanently left with **-2** in everything.

# Steps To Reproduce

1. Consume Panaceus.
2. Wait for its effect to pass (only takes a few turns).
3. Witness multiple, devastating and permanent stat reductions.

# Expected behavior

I'm not certain what the intended effect for Panaceus is exactly, but its description hints to something vaguely positive. That please.

# Screenshots

Panaceus active effect, notice how the advertised +2 is not applied:
![pana](https://user-images.githubusercontent.com/45204409/92062899-0e5ddb80-ed9a-11ea-8bce-f9a3e3577777.png)

After the effect is over:
![pana2](https://user-images.githubusercontent.com/45204409/92062923-1ddd2480-ed9a-11ea-8cf0-b80ae34fb988.png)

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.E-5620-gc4627a8 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Blaze Industries [blazeindustries],
    C.R.I.T Expansion Mod [crt_expansion]
]

# Additional context

~~Please tell me how to debug my character, if it's possible.~~

Edit: After 2 in-game hours I only have -1 in all stats, the negative effect seems to go away slowly, I'm not sure anymore, is this effect intended?

**Comment by Valares** [comment]
I've created a character based on your data. I've used the same version and all mods, you have, added the traits you have and spawned in a Panaceus.

I get a +2 bonus to all stats. I can not confirm your bug.

However, I can also see on your screenshot that you have a 15% speed reduction caused by pain. After adding 50 pain to myself, I had the same speed malus and malus on all stats with the exception of Perception (in line with what it says when visiting the "pain" effect).
I guess the negative effects on your characters stats comes from the pain they have, and the bonus of the Panaceus did counter it for a bit.

By the way, the positive effect on the stats is just a side effect.
<details>
  <summary>The true purpose of Panaceus (spoiler)</summary>
  It's true purpose is to heal almost all illnesses instantly, ranging from poison over parasites, infections up to flu/colds.
</details>
